google/cloud/spanner_dbapi/partition_helper.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+from typing import Any
+
+import gzip
+import pickle
+import base64
+
+
+def decode_from_string(encoded_partition_id):
+    gzip_bytes = base64.b64decode(bytes(encoded_partition_id, "utf-8"))
+    partition_id_bytes = gzip.decompress(gzip_bytes)
+    return pickle.loads(partition_id_bytes)
+
+
+def encode_to_string(batch_transaction_id, batch_result):
```

Comment:
What is `batch_result` in this case? The partition ID? If so, I think it would be better to name it like that. This sounds more like a query result.

Reply:
Renamed to partition_result as it contains the partition details coming from Spanner